changelogs/internal/newsfragments/1713.misc

```diff
@@ -0,0 +1 @@
+Fix npm release script for `@matrix-org/spec`.
```

Comment:
somehow both CI and I missed this in review: `.misc` doesn't exist in the spec - we use `.clarification`